src/sdk/pynni/nni/evolution_tuner/parameter_expressions.py

```diff
@@ -65,7 +65,7 @@ def loguniform(low, high, random_state):
     high: an float that represent an upper bound
     random_state: an object of numpy.random.RandomState
     '''
-    return np.exp(uniform(low, high, random_state))
+    return 10 ** (uniform(np.log10(mu), np.log10(sigma), random_state))
```

Comment:
what is mu and sigma?

Reply:
Sorry I forgot to modify the variable name... Will fix, thanks~